.circleci/config/base.yml

```diff
@@ -2291,6 +2291,7 @@ jobs:
       <<: *env-stack-dumping
     parallelism: 2
     steps:
+      - run: nvm install lts/iron && nvm alias default lts/iron
```

Comment:
@MarshallOfSound we need Node >= 18 for our test dependencies, https://app.circleci.com/pipelines/github/electron/electron/80090/workflows/eb28bb2e-3618-46aa-ab0a-2d45f4cd34a4/jobs/1699689. Added this temporary to unblock the roll.